pandas/tests/sparse/test_frame.py

```diff
@@ -1118,6 +1120,28 @@ def test_isnotnull(self):
                             'B': [True, False, True, True, False]})
         tm.assert_frame_equal(res.to_dense(), exp)
 
+    @pytest.mark.importorskip('scipy.sparse')
```

Comment:
actually on 2nd thought, use the ``tm._skip....`` as we are consistently using that. (does the same thing)